### DISCUSSION TITLE
UART Setup

### DISCUSSION THREAD

**Discussion by hansemro** [hansemro/kc1-linux#17] [Show and tell]
Here are some photos:
![image](https://user-images.githubusercontent.com/40348686/115506870-3e29f880-a230-11eb-8dc1-02d89955f687.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115507365-e213a400-a230-11eb-9a34-01fae61f08d2.jpg)
![image](https://user-images.githubusercontent.com/40348686/115506925-4f730500-a230-11eb-863b-ee14eb7a5cbe.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115506982-5dc12100-a230-11eb-8851-58a74d507394.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115507012-67e31f80-a230-11eb-9891-5cdcaa9038f4.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115507030-716c8780-a230-11eb-99fd-36eda32d0687.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115507072-7cbfb300-a230-11eb-866b-0abf0fd02465.jpeg)
![image](https://user-images.githubusercontent.com/40348686/115507093-86491b00-a230-11eb-9fff-2c6397a9248e.jpeg)

**Comment by hansemro** [comment]
Current setup:
- 30 AWG wires
- Sparkfun FT232RL Breakout: https://www.sparkfun.com/products/12731
- Sparkfun Logic Level Converter Breakout: https://www.sparkfun.com/products/14765

Notes: Solder pads are very weak! I already messed up 2 boards because of my impatience and poor solder job. Secure the wires afterwards (with hot glue, tape, etc.).

**Reply by mj-sakellaropoulos** [reply]
I've used magnet wire for my mod as recommended by Louis Rossman. Heat controlled soldering iron and lots of flux is key for boards like these.

**Reply by hansemro** [reply]
Thanks for the recommendation

**Reply by hansemro** [reply]
I actually did try with wires I ripped from an inductor since I did not have thin wires at the time. However that board died before I could actually test it.

**Comment by hansemro** [comment]
Unrelated: Here it is running a simple C ARMv8 Simulator I wrote for an assignment (3.0.72 kernel + display + touchscreen + WiFi):

https://user-images.githubusercontent.com/40348686/115509673-8d255d00-a233-11eb-8db5-a18e6a41bf6a.mp4

**Comment by mj-sakellaropoulos** [comment]
I added a header a couple years back, now the back cover doesent go back on completely 😅

![image](https://user-images.githubusercontent.com/36081545/115822347-65b8c680-a3d2-11eb-8b26-3f4115da263b.png)

**Reply by hansemro** [reply]
I am tempted to add headers on mine now.

**Comment by hansemro** [comment]
Alternative UART pins:
UART3_TX: pin 13
UART3_RX: pin 14
![AltUART](https://user-images.githubusercontent.com/40348686/161001631-90f8caab-8236-4004-9267-63cf1c16f286.png)
